src/legacy/core_plugins/vis_default_editor/public/components/agg_params.tsx

```diff
@@ -75,16 +78,18 @@ function DefaultEditorAggParams({
   onAggTypeChange,
   setTouched,
   setValidity,
+  schemas,
+  allowedAggs,
 }: DefaultEditorAggParamsProps) {
-  const groupedAggTypeOptions = useMemo(() => getAggTypeOptions(agg, indexPattern, groupName), [
-    agg,
-    indexPattern,
-    groupName,
-  ]);
+  const groupedAggTypeOptions = useMemo(
+    () => getAggTypeOptions(agg, indexPattern, groupName, allowedAggs || []),
+    [agg, indexPattern, groupName, allowedAggs]
```

Comment:
we need to pass allowedAggs in as that information is no longer available on the schema